src/gui/input.cpp

```diff
@@ -164,8 +164,11 @@ QString convertKey(const QString& text, int key, Qt::KeyboardModifiers mod) noex
 		return ToKeyString(GetModifierPrefix(mod), specialKeys.value(key));
 	}
 
+	// The key "<" should be sent as "<lt>"
+	// Issue#607: Remove ShiftModifier from "<", shift is implied.
 	if (text == "<") {
```

Comment:
I wonder if should apply to other modifiers too. Alt/Cmd/Ctrl.

Reply:
I'm not sure I follow... Could you elaborate?

Reply:
Seems like pressing Alt+<, or Ctrl+<, also generate \<lt\> in terminal.

Having said that these things are keyboard layout specific, e.g. in some keyboard layouts < does not require a Shift key but \<gt\> does.

Reply:
Ahhhh, I understand.

I can get `Alt + <` to map in the TUI:
```
:nn <A-lt> :echo 'Alt + "<"'<CR>
{ Press Shift + Alt + ,< }
{ Displays: Alt + "<"}
```

It sounds like `<S-lt>` is the only combination that should not appear.

Reply:
> I can get Alt + < to map in the TUI:

Interesting, I cant. Carry on then.